x-pack/solutions/security/packages/features/src/security/v2_features/kibana_sub_features.ts

```diff
@@ -674,23 +670,25 @@ const endpointExceptionsSubFeature = (): SubFeatureConfig => ({
       privileges: [
         {
           id: 'endpoint_exceptions_all',
-          includeIn: 'all',
+          includeIn: 'none',
```

Comment:
Just clarifying this and the below `none` is to not introduce a breaking change to serverless?

Reply:
Right, rolled back